src/Type/Definition/NonNull.php

```diff
@@ -4,12 +4,21 @@
 
 namespace GraphQL\Type\Definition;
 
+use GraphQL\Error\InvariantViolation;
+use GraphQL\Type\Schema;
+use function is_callable;
+
 class NonNull extends Type implements WrappingType, OutputType, InputType
 {
-    /** @var NullableType&Type */
+    /** @var callable|(NullableType&Type) */
     private $ofType;
 
-    public function __construct(NullableType $type)
+    /**
+     * code sniffer doesn't understand this syntax. Pr with a fix here: waiting on https://github.com/squizlabs/PHP_CodeSniffer/pull/2919
+     * phpcs:disable
```

Comment:
can we disable only incompatible sniff, not all?

Reply:
Sure. cfb3d3a